coderd/workspaces.go

```diff
@@ -136,7 +136,7 @@ func (api *API) workspace(rw http.ResponseWriter, r *http.Request) {
 // @Security CoderSessionToken
 // @Produce json
 // @Tags Workspaces
-// @Param q query string false "Search query in the format `key:value`. Available keys are: owner, template, name, status, has-agent, dormant, last_used_after, last_used_before."
+// @Param q query string false "Search query in the format `key:value`. Available keys are: owner, template, name, status, has-agent, dormant, last_used_after, last_used_before, has-ai-task."
```

Comment:
Damn, seems like we already have two naming conventions (`has-agent`, `last_used_after`) 😞
I think we should support both by replacing the separators with one or the other.
Could be in a separate PR though, or just turned into an issue since this isn't your fault.

Reply:
Created an issue for this: https://github.com/coder/coder/issues/18429